Gems/SceneProcessing/Code/Tests/SceneBuilder/SceneBuilderConfigTests.cpp

```diff
@@ -159,3 +191,145 @@ TEST_F(SceneProcessingConfigTest, SceneProcessingConfigSystemComponent_ScriptCon
         EXPECT_TRUE(result.has_value());
     sceneProcessingConfigSystemComponent.Deactivate();
 }
+
+TEST_F(SceneProcessingConfigTest, SceneProcessingConfigSystemComponent_SoftNameSettings_MatchesSettingRegistry)
+{
+    const char* settings = R"JSON(
+    {
+        "O3DE": {
+            "AssetProcessor": {
+                "SceneBuilder": {
+                    "NodeSoftNameSettings": [
+                        {
+                            "pattern": {
+                                "pattern": "^.*_[Ll][Oo][Dd]_?1(_optimized)?$",
+                                "matcher": 2
+                            },
+                            "virtualType": "LODMesh1",
+                            "includeChildren": true
+                        }
+                    ],
+                    "FileSoftNameSettings": [
+                        {
+                            "pattern": {
+                                "pattern": "_anim",
+                                "matcher": 1
+                            },
+                            "virtualType": "Ignore",
+                            "inclusiveList": false,
+                            "graphTypes": {
+                                "types": [
+                                    {
+                                        "name": "IAnimationData"
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            }
+        }
+    }  
+    )JSON";
+    m_settingsRegistry->MergeSettings(settings, AZ::SettingsRegistryInterface::Format::JsonMergePatch);
+
+    AZStd::unique_ptr<AZ::SerializeContext> serializeContext = AZStd::make_unique<AZ::SerializeContext>();
+    AZStd::unique_ptr<AZ::JsonRegistrationContext> registrationContext = AZStd::make_unique<AZ::JsonRegistrationContext>();
+    m_settingsRegistry->SetContext(serializeContext.get());
+    m_settingsRegistry->SetContext(registrationContext.get());
+    ReflectTypes(registrationContext.get(), serializeContext.get());
+
+    AZ::SceneProcessingConfig::SceneProcessingConfigSystemComponent sceneProcessingConfigSystemComponent;
+    sceneProcessingConfigSystemComponent.Activate();
```

Comment:
suggestion: I wonder if some of this code that's repeated in each test could either be moved to the fixture setup/teardown or just a separate function (or member function) we could call to reduce the duplication